LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 RudderStack
+Copyright (c) 2021 Grafbase
```

Comment:
Should this be changed?

Reply:
MIT is subject to the condition that
> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

So changing it seems bad?

Reply:
Ok changed it back